webapp/login/views.py

```diff
@@ -96,6 +105,72 @@ def after_login(resp):
     return flask.redirect(open_id.get_next_url())
 
 
+@login.route("/login-candid", methods=["GET"])
```

Comment:
```suggestion
@login.route("/login-beta", methods=["GET"])
```

I think we can name it beta. Users don't need to know the tool that is running in the backend.

Reply:
Makes sense, done :)